### ISSUE TITLE
Gender does not get set when creating new contact

### ISSUE THREAD

**Issue by vspaziani** [monicahq/chandler#500]
When adding a new contact through the contact form, if the gender is added it does not appear as set in the contacts display
<img width="484" alt="Screenshot 2023-05-08 at 7 25 52 PM" src="https://user-images.githubusercontent.com/32283373/236971033-e45495cc-e6d7-4309-95db-f0c8c61c3d15.png">
<img width="1149" alt="Screenshot 2023-05-08 at 7 26 18 PM" src="https://user-images.githubusercontent.com/32283373/236971035-646e77ec-bac1-4be7-a943-11c5952275d6.png">

**Comment by djaiss** [comment]
What's your browser? 
It works on the `main` branch on Firefox and Chrome.

**Comment by vspaziani** [comment]
You are correct, I just tested it on the same computer. It sets correctly when using chrome. I have Safari Version 16.4 Mac OS but the same behavior exists on IOS. I did some additional troubleshooting. On Safari, the selection appears to be already made as shown in my screen shot denoted by the check mark. (please note that I have not actually selected anything yet but male is checked)
<img width="460" alt="Screenshot 2023-05-09 at 9 18 29 PM" src="https://github.com/monicahq/chandler/assets/32283373/4569a336-5b65-4569-83f8-2666cff8d07d">



 But if I  choose female or actually click on male (even though it is already checked)  then save the new contact the gender is correctly saved. In chrome, the gender is not pre-populated with anything so you have to actually make a selection. I have attached two screen shots, in both I have not actually made a selection yet.

This one is from Chrome:
<img width="503" alt="Screenshot 2023-05-09 at 9 14 35 PM" src="https://github.com/monicahq/chandler/assets/32283373/831481b3-bcb5-4d62-9c4d-a69a89096347">

This one if from Safari:
<img width="482" alt="Screenshot 2023-05-09 at 9 17 06 PM" src="https://github.com/monicahq/chandler/assets/32283373/cff8d29a-e4c6-4b15-a3c0-95da2c0cbee7">

**Comment by djaiss** [comment]
Thank you for the comprehensive response. 

Unfortunately, this is the standard behavior of Safari. I'm not sure if we can do anything about it.
